frontend/src/components/Output/AnalysisDeviance/ContinuousDeviance.js

```diff
@@ -0,0 +1,43 @@
+import React, {Component} from "react";
+import {observer} from "mobx-react";
+import PropTypes from "prop-types";
+import {ff} from "../../../common";
+@observer
```

Comment:
formatting; linebreak between imports and code

Reply:
fixed